tests/testthat/test-unnecessary_lambda_linter.R

```diff
@@ -34,28 +34,36 @@ test_that("unnecessary_lambda_linter skips allowed usages", {
     NULL,
     linter
   )
+
+  # This _could_ be lapply(x, `%in%`, tbl), but don't force infix into lambda
+  expect_lint("lapply(x, function(xi) xi %in% tbl)", NULL, linter)
+  # This one could not
+  expect_lint("lapply(x, function(xi) tbl %in% xi)", NULL, linter)
```

Comment:
This one could also, using a named arg, be ``lapply(x, `%in%`, x = tbl)``.

Reply:
even harder to read, but indeed :)